src/hci/objects_stats.cpp

```diff
@@ -170,6 +170,8 @@ void StatsFormButton::addCostBar()
 	sBarInit.iRange = GAME_TICKS_PER_SEC;
 	attach(costBar = std::make_shared<W_BARGRAPH>(&sBarInit));
 	costBar->setBackgroundColour(WZCOL_BLACK);
+	// set a custom hit-testing function that ignores all mouse input / clicks
+	costBar->setCustomHitTest([](WIDGET *psWidget, int x, int y) -> bool { return false; });
```

Comment:
BTW, in the PR https://github.com/Warzone2100/warzone2100/pull/2079/files I added a function setTransparentToMouse, that should have the same effect, I believe.